packages/dom/test/unit/lib/ElementAssertion.test.tsx

```diff
@@ -172,4 +173,101 @@ describe("[Unit] ElementAssertion.test.ts", () => {
       });
     });
   });
+
+  describe(".toHaveClass", () => {
+    context("when the element has the expected class", () => {
+      it("returns the assertion instance", async () => {
+        const { findByTestId } = render(<HaveClassTestComponent />);
+        const divTest = await findByTestId("classTest");
+        divTest.classList.add("foo", "bar");
```

Comment:
Maybe we can pass a `className` prop to the `<HaveClassTestComponent />` component to avoid using `classList.add(..)`, wdyt?